Simplified/AxisService/NYPLAxisDRMAuthorizer.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  NYPLAxisDRMAuthorizer.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-13.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+#if AXIS
+@objcMembers
+class NYPLAxisDRMAuthorizer: NSObject, NYPLDRMAuthorizing {
+  
+  static let deviceIDKey = "NYPLAxisDRMAuthorizerKey"
+  
+  static let sharedInstance = NYPLAxisDRMAuthorizer()
+  
+  static var deviceID: String {
+    if
+      let id = UserDefaults.standard.string(forKey: NYPLAxisDRMAuthorizer.deviceIDKey) {
+      return id
+    }
+    
+    let id = UUID().uuidString
+    UserDefaults.standard.set(id, forKey: NYPLAxisDRMAuthorizer.deviceIDKey)
+    return id
+  }
+  
+  /// TODO: To be fixed in SIMPLY-3692. 
+  var workflowsInProgress: Bool {
+    return false
+  }
+  
+  /// We're always returning true since Axis does not deauthorize a user.
+  func isUserAuthorized(_ userID: String!,
+                        withDevice device: String!) -> Bool {
+   return true
+  }
+  
+  /// TODO: SIMPLY-3692 will have code that fixes the issue.
```

Comment:
sorry, which issue are you referring to? 

regarding possibly clearing the deviceID from UserDefaults, I sort of think we don't have to worry about that. Even if the user logs out, I don't see any issues with just leaving it there. It might actually be better to leave it, so it's consistently unique per device (unless user uninstalls the app, but that another story, and it's unavoidable)

Reply:
Great catch. This is leftover comment from when we were not sure what to do with deviceID.  Removed.